src/devices_models/devices/hydro_generation.jl

```diff
@@ -753,9 +751,9 @@ active power budget formulation.
 function energy_budget_constraints!(
     optimization_container::OptimizationContainer,
     devices::IS.FlattenIteratorWrapper{H},
-    model::DeviceModel{H, <:AbstractHydroFormulation},
-    system_formulation::Type{<:PM.AbstractPowerModel},
-    feedforward::Union{Nothing, AbstractAffectFeedForward},
+    ::DeviceModel{H, <:AbstractHydroFormulation},
```

Comment:
Do we need to put this in the style guide? I'm OK with it. Some people prefer including the name because it does inform the reader about what is being passed even if it is unused. That isn't too important if the type isn't specified but there is at least one example here where the type is not specified.

Consistency is better than anything.

Reply:
we could add it to the style guide. To me, it makes it easier to read @code_warntype returns